pkgs/by-name/no/notion-app/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
```

We can use `lib.importJSON`. (Also, it's `hostPlatform`, not `targetPlatform`!)
```suggestion
    (lib.importJSON ./info.json)."${stdenvNoCC.hostPlatform.system}"
```